src/index.tsx

```diff
@@ -93,6 +97,7 @@ const TextareaAutosize: React.ForwardRefRenderFunction<
   if (typeof document !== 'undefined') {
     React.useLayoutEffect(resizeTextarea);
     useWindowResizeListener(resizeTextarea);
+    useFontsLoadedListener(resizeTextarea);
```

Comment:
To save some bytes I would prefer to have a single generic hook that would be able to handle both of those.

You can copy-paste my implementation from [here](https://www.typescriptlang.org/play?#code/JYWwDg9gTgLgBAVwM4FMAyBDGKnwGZQQhwBEyKAtADZY4wkDcAUKJLHAEooYDG+hxElG59GTJjACeYFHACqAOwDWCiAHcFAMQQK+wCArgBeOAAoAdJYxQA5kgBccDAskBtALoBKYwD4nL5glpWQBJBTwUKABRADcUBRgAFWCAHkTE6xsUGD8TdMzsuBQAD2wFABMkOABvJjg4AHoGuDVZYHDIuAIiOAAmOAg4qCoIDEqWgAtgHgmnYTgkBBkoOB4iEAMu6CK4hLgYApgq9v2J2QARAHkAWTgqYAAjOsbm1tIwKH1PmGAALxQSKdZHhgFBcAMFLIMFUYGcuqDwQY2lUMFQ1BhJFUNvMkDIeMAQTxnmNyrF4jA0MBcPFIqYpDJHO0IisAAoAGjglnMmQc-kknkcMQgwHKzHqTRash4zjgwBsqnmsNkqDWFQhUOOMAA5FVkAhUVRJHAAFbIeAYLoGh68JSnLBOKgjNRVa3zAAGuE+ChsbrgZ2Ezwlb3IcDdTM6LN6vsGnTdzkkvuhkrgwgNRvKBi18Gl8wwDwgCHgsKp+2CcBiqIQKEDzWc5WTGYUWZaziLECcPHx5XJactwCoTlLMicSCQcoUef7wCkqzOPCUHOEIBQIAenVh9rjLl9VKb5tH47zVFkMHbbshQzdxPKpN2FKpZVp9JQjI6rN6HK5PMc8YF5eFopMAAvs8AD8cAsnAABkCwwF6NjPI4F6RIEz5wGEzJkgkKTPPktjZGyuHJMOJRlOMnrtAhuShgYAAk1TpMEQG+qR8TjEoKCSBAeBwHhWQwGBEHWBgy7YGCKRRKUUC8DAaQZPhMCuG6dEMcRKDMe4HKKCo6haDoegGD4PiuAADO4iFwFhMCBHg+k-Js5CUtSkJQDh9R8YUrEVFUVnyfxhHuWpRSlGxVQYZEvmpB5ORMD4pjPAcCmONFAVDi+vFqal9zOZEjimCgd6OOF0R3nJhwcoxMg+N4Rh+EKIpMN4tT1Kq4I0NguBOY+KwmI5tC4KY2XdZ4YqcCIMDmH1XGFlEeARHwpimDVfjNfUqwGOCEx1seUCOM+3F3A+NI9WYBXLXc-X3jlUDmDwCBQMICT5TEI3iGt+yHNyN5WV1x10sEHJbRUO2ve9wgwPdhhLb4H0KeYS4xj9R0uf9MiA9tkSg3AQEch4r0gUwjnI7SajtBmagckIOB-ACHLPedq0FTWcAAHqgcBr1E9dpgZndy4JOYeAGEclMjGMlGNrTp0vTDjMxMzbMc8wQA)